website/static/js/contribAdder.js

```diff
@@ -337,7 +320,10 @@ var AddContributorViewModel = oop.extend(Paginator, {
         return $osf.postJSON(
             nodeApiUrl + 'contributors/', {
                 users: self.selection().map(function(user) {
```

Comment:
For browser compatibility, can you change this to use `ko.utils.arrayMap` or `$.map`?